<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQ Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="calc_SQ_array_checkBox">
      <property name="text">
       <string>Calculate SQ limit for an array</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="load_pushButton">
      <property name="text">
       <string>Load SMARTS Spectrum</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="load_checkBox">
      <property name="text">
       <string>Use ASTMG173</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Bandgap (eV)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="bandgap_min_doubleSpinBox">
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Temperature (K)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="bandgap_max_doubleSpinBox">
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="temp_spinBox">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="bandgap_doubleSpinBox">
      <property name="value">
       <double>1.630000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Bandgap MAX (eV)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="plot_checkBox">
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Bandgap MIN (eV)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="CalcualteSQ_pushButton">
      <property name="text">
       <string>Calculate SQ Limit</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="save_pushButton">
      <property name="text">
       <string>Save Results for Bandgap array</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="10">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
